build.assets/Dockerfile

```
@@ -298,7 +298,7 @@ RUN go install github.com/google/addlicense@v1.0.0
 RUN go install github.com/daixiang0/gci@v0.9.1
 
 # Install golangci-lint.
-RUN curl -fsSL "https://raw.githubusercontent.com/golangci/golangci-lint/v1.52.2/install.sh" | \
+RUN curl -fsSL "https://raw.githubusercontent.com/golangci/golangci-lint/v1.53.0/install.sh" | \
     sh -s -- -b "$(go env GOPATH)/bin"
```

As discussed in Slack, this will actually pull the latest version.
The recommended way in [golangci docs](https://golangci-lint.run/usage/install/#binaries) is to
```shell
curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.52.2
```

However, using `master` does not sound like a great idea.
I think we should pin both the github tag and the binary version.
Something like this:
```suggestion
curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/v1.53.0/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.53.0
```

---

Done in cbb6eb47f74b974d221b011c7a8cd5597feffc44